features/step_definitions/all_step_definitions.rb

```diff
@@ -1,16 +1,29 @@
 Given("I am on the landing page") do
   visit root_path
 end
+
+Given("the following user exist") do |table|
+  table.hashes.each do |person|
+  User.create!(person)
```

Comment:
Indentation

Reply:
Please do not click "Resolve" on a comment without actually addressing the issue.